utils/grpc/setup_server.go

```diff
@@ -52,7 +53,10 @@ func (s *GrpcServer) GrpcServer(port string, dbURL string, rdURL string, server
 
 	s.serverTypeToServerName(server)
 
-	s.dBURL = dbURL
+	//connections delimited by ','
+	dbURList := strings.Split(dbURL, ",")
```

Comment:
maybe you should change the dbUrl to ...string type, so far I believe split with ',' should be fine, but still can not fully sure, thinking about it password contains ',' .

Reply:
@taiyangc thoughts on this ?

Reply:
I am confused as to how this is getting used? Are these multi dbs getting load balanced through the url list? Or is it used like the previous PR, one db for each usage. If one db for each type of usage, how do you clearly define the ordering within the list? Not very easy to track.

Reply:
I may need to understand the initial insight on making this a multi db list.

Reply:
In my mind, it is similar to the previous PR, one db for each usage. As for the multi db list, I would imagine the user of this setup would pass in a comma delimited string (database connections strings), where it will be used by, for example, checkruntime, to find the status/health of each database. Maybe the list isn't the way to go, but maybe I need a better understanding of the intent and direction. @wxue

Reply:
I see. It makes sense to fix on a format for check runtime to work on multiple urls. However, the passed in information should include information about its main usage. Right now our variable names are meaningful, e.g. `DB_URL` or `GUARD_DB_URL`, maybe a prefix is needed in this case to have `DB_URL_STATUS`, `DB_URL_GUARD` etc. so we can clearly pass in from devops/build, and also checked by runtime with a clear list. thoughts @wxue @uchetron ?

Reply:
@taiyangc I agree. Along those lines, would an approach be the following: there should be some *information* (key) about the usage for *each url* (value) in the string, especially for tracking purposes. For example, var dbUrls = " DB_URL_STATUS : postgresql://xdsds:asdss!@localhost:5432/db_status , DB_URL_GUARD : postgresql://xdsds:313sdsds!@localhost:5432/db_guard " passed in to the GrpcServer(..) function. Thoughts ? @taiyangc @wxue

Reply:
**Context:** 
Due to one service may need multiple DB(different instance) access, on the one hand, we need set e.g DB_URL_STATUS and DB_URL_GUARD for different db as env var so it can be pass in from devops. On the other hand, gprc server set up checks and runtime checks need to check multiple/all the DB as well, so the PR should handle the checking part.

**Thoughts:**
Here're some of the possible solutions in my mind:

1. if we strictly require backward support(keep the dbUrls a string):
    * split multi db URL string by some special character
    * convert db URLs map into byte string then convert back (and add backward checks on the format)

2. if we to handle this more straightforward and make it easier in the future (breaking change):
   * change dbUrls input as a []string, and get/check each db one by one
   * change dbUrls input as a map[string]string

Hope any of those make sense and preferred to you.

@uchetron @taiyangc @JustinGaoF

Reply:
Hey, @taiyangc Do we need backwards compatibility ? I think maybe it is best to go with (2) map[string]string this is sort of what I had in mind, since it keeps things simple, yet organized and trackable, give the key, i think, but breaks input types of setupServer. Thoughts ?

Reply:
@uchetron Yes. So let's create a new variable that can take this map format. Thanks. Be careful that the passwords within postgres url can contain any special character.. so the split should probably on a known prefix such as `postgresql://`